packages/web-worker/README.md

```diff
@@ -0,0 +1,47 @@
+# @bugsnag/web-worker
+
+This package contains the web worker / service worker implementation of the BugSnag library.
+
+## Features and limitations
+
+This early release offers basic functionality for web workers and service workers, with the intent to support Chrome extension development using manifest v3. Using this library, you will be able to:
+
+- notify errors from within service workers and web workers, including browser extensions
+- detect and automatically notify unhandled errors from service workers and web workers, excluding Chrome browser extensions due to limitations in the Chrome Runtime API.
+
+## Installation
+
+```bash
+npm i @bugsnag/web-worker@next
```

Comment:
This won't need the `@next` when it's released, or are we planning another prerelease before then?

```suggestion
npm i @bugsnag/web-worker
```